pkgs/tools/security/snow/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "snow";
+  version = "20130616";
+
+  makeFlags = [ "CFLAGS=-O2" ];
```

Comment:
Can you move this under the fetchurl? Then it follow a more logical order.

Reply:
Ok